lib/util/cache/repository/index.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { mocked } from '../../../../test/util';
 import { GlobalConfig } from '../../../config/global';
-import type { RenovateConfig } from '../../../config/types';
+import type { WorkerPlatformConfig } from '../../../workers/repository/init/apis';
```

Comment:
this is a bad idea, we don't like to import types from worker to utils.

we should define what we need here instead